guide/index.html

```diff
@@ -0,0 +1,230 @@
+    <meta property="og:image:height" content="630" />
+    <meta property="og:image:alt" content="An illustrated image of Earth in space, surrounded by a green background, and text: ClimateAction.Tech; Technology professionals using our skills, tools, and influence to amplify the Climate Movement" />
+    <meta property="og:type" content="website" />
+    <meta property="og:description" content="Technology professionals using our skills, tools, and influence to amplify the Climate Movement" />
+
+    <!-- Twitter Metadata -->
+    <meta name="twitter:card" content="summary" />
+    <meta name="twitter:site" content="ClimateAction.tech" />
+    <meta name="twitter:title" content="ClimateAction.tech" />
+    <meta name="twitter:description" content="Technology professionals using our skills, tools, and influence to amplify the Climate Movement." />
+    <meta name="twitter:image" content="https://climateaction.tech/img/earth.png" />
+
+    <title>ClimateAction.tech</title>
+
+    <!-- Bootstrap Core CSS -->
+    <link href="/vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">
+
+    <!-- Theme CSS -->
+    <link href="/css/freelancer-3.css" rel="stylesheet">
+
+    <!-- Custom Fonts -->
+    <link href="vendor/font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Lato:400,700,400italic,700italic" rel="stylesheet" type="text/css">
+
+    <!-- Favicon -->
+    <link rel="icon" type="image/png" href="/img/earth.png" />
+
+    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
+    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
+    <!--[if lt IE 9]>
+        <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
+        <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
+    <![endif]-->
+
+</head>
+
+<body id="page-top" class="index">
+<div id="skipnav"><a href="#maincontent">Skip to main content</a></div>
+
+    <!-- Navigation -->
+    <nav id="mainNav" class="navbar navbar-default navbar-fixed-top navbar-custom">
+        <div class="container">
+            <!-- Brand and toggle get grouped for better mobile display -->
+            <div class="navbar-header page-scroll">
+                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
+                    <span class="sr-only">Toggle navigation</span> Menu <i class="fa fa-bars"></i>
+                </button>
+                <a class="navbar-brand" href="#page-top">ClimateAction.tech</a>
+            </div>
+
+            <!-- Collect the nav links, forms, and other content for toggling -->
+            <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
```

Comment:
Given that this this is looking in okay shape now, how about we add it to the menu? There's no way to access this guide page without knowing the `url` in advance.

Reply:
Done in ef84ca0